src/diffusers/pipelines/ltx/pipeline_ltx_condition.py

```diff
@@ -447,6 +447,16 @@ def _pack_latents(latents: torch.Tensor, patch_size: int = 1, patch_size_t: int
         post_patch_num_frames = num_frames // patch_size_t
         post_patch_height = height // patch_size
         post_patch_width = width // patch_size
+
+        latent_sample_coords = torch.meshgrid(
+            torch.arange(0, num_frames, patch_size_t, device=device),
+            torch.arange(0, height, patch_size, device=device),
+            torch.arange(0, width, patch_size, device=device),
+        )
```

Comment:
```suggestion
        latent_sample_coords = torch.meshgrid(
            torch.arange(0, num_frames, patch_size_t, device=device),
            torch.arange(0, height, patch_size, device=device),
            torch.arange(0, width, patch_size, device=device),
            indexing="ij",
        )
```

https://pytorch.org/docs/stable/generated/torch.meshgrid.html

Default `indexing` in PyTorch will change from `ij` (same as numpy) to `xy`, there's a warning if we don't specify `indexing="ij"`.